lib/src/main/java/space/taran/arklib/domain/score/ScoreStorageRepo.kt

```diff
@@ -0,0 +1,36 @@
+package space.taran.arklib.domain.score
+
+import kotlinx.coroutines.CoroutineScope
+import space.taran.arklib.domain.index.ResourceIndex
+import space.taran.arklib.domain.index.RootIndex
+import java.nio.file.Path
+
+class ScoreStorageRepo(private val scope: CoroutineScope) {
+    private val storageByRoot = mutableMapOf<Path, RootScoreStorage>()
+
+    suspend fun provide(index: ResourceIndex): Any {
```

Comment:
`Any` is bad practice, isn't there a suitable interface?

Reply:
It's unchecked copy-paste. Or something random. Must be fixed for sure.